lms/views/api/d2l/files.py

```diff
@@ -29,15 +30,37 @@ def list_files(_context, request):
     permission=Permissions.API,
 )
 def via_url(_context, request):
+    course_copy_plugin = request.product.plugin.course_copy
```

Comment:
This is very similar to the Blackboard version. I think we'll have an opportunity to refactor this when we refactor the Canvas version.